archive/app/ui/NotFound.scala

```diff
@@ -1,13 +1,18 @@
 package ui
 
-import conf.Configuration
+import conf.{Static, Configuration}
 import play.api.libs.json.{JsValue, Json}
 import rendering.Renderable
+import conf.switches.Switches.{PolyfillIO}
 
 object NotFound extends Renderable {
+  val polyfillioUrl = if(PolyfillIO.isSwitchedOn) common.Assets.js.polyfillioUrl else Static("javascripts/vendor/polyfillio.fallback.js")
+
   override def props: Option[JsValue] = Some(
     Json.obj(
-      "beaconUrl" -> Configuration.debug.beaconUrl
+      "beaconUrl" -> Configuration.debug.beaconUrl,
+      "bundleUrl" -> Static("javascripts/ui.bundle.browser.js"),
```

Comment:
Would this bundleUrl be the same for all `Renderable`? If yes can we move this to a more central place

Reply:
@TBonnin yes and the logic for polyfillioUrl will also be common across all renderables. I can extract to a reusable/central resource.